reliquery/relic.py

```diff
@@ -247,6 +250,15 @@ def get_image(self, name: str) -> BytesIO:
         self.assert_valid_id(name)
         return self.storage.get_binary_obj([self.relic_type, self.name, "images", name])
 
+    def display_image(self, name: str) -> Image:
+        self.assert_valid_id(name)
+        img_data = self.storage.get_binary_obj(
+            [self.relic_type, self.name, "images", name]
+        )
+        img_bytes = img_data.read()
```

Comment:
is img_data a BytesIO object? if so we don't need to read it out we can pass it in the image open method.